roles/static_inventory/defaults/main.yml

```diff
@@ -19,3 +19,6 @@ private_ssh_key: ~/.ssh/openshift
 
 # The patch to store the generated config to access bastion/hosts
 ssh_config_path: /tmp/ssh.config.ansible
+
+openstack_region: regionOne
```

Comment:
The users already specify the region in their openstack credentials (alongside the username, tenant, etc.) so we should just read the value from there. Having to enter the same value twice in different places will only lead to confusion.

See: http://docs.ansible.com/ansible/latest/os_client_config_module.html

I've tested the above with `openstack.clouds[0].region_name` and it seems to always work so let's use that instead.